<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="qt_groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>We are here in Qt</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="sendToCSharp_pushButton">
       <property name="text">
        <string>Send to C#</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="sendToCSharp_lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>From C#</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="fromCSharp_lineEdit_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="csharp_groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>This is the C# visitor</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
